Services/OmniSearch.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Leantime\Plugins\OmniSearch\Services;
+
+class OmniSearch {
+
+  public function install() {
+    $omniSelectEngine = file_get_contents(dirname(__DIR__). "/assets/js/omniSelectEngine.js");
+    $fp = fopen($_SERVER['DOCUMENT_ROOT'] . "/dist/js/omniSearch.js", "wb");
+    fwrite($fp,$omniSelectEngine);
```

Comment:
Why not just use `https://www.php.net/manual/en/function.symlink.php` so updates to the module is applied by default... and you could remove the part about symlinks in the readme?